main.py

```diff
@@ -20,10 +20,11 @@ def get_cat_facts():
     resp = requests.get("https://catfact.ninja/facts")
 
     # I believe this allows 200-series codes & 300-series codes
-    try: resp.raise_for_status()
+    try:
+        resp.raise_for_status()
     except HTTPError:
```

Comment:
Are you sure this is working? I don't see this imported. Please test running this API